ui/app/src/services/content.ts

```diff
@@ -1044,3 +1038,15 @@ export function uploadDataUrl(
     });
   });
 }
+
+export function getQuickCreateContentList(siteId: string) {
```

Comment:
Add to the default export below.
Make sure to pull once https://github.com/craftercms/studio-ui/pull/1634 is merged. There's a bunch more on the default export list.